com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -109,6 +109,7 @@ public Matrix4x4 GetGPUProjectionMatrix(int viewIndex = 0)
         public CameraType cameraType;
         public bool isDefaultViewport;
         public bool isHdrEnabled;
+        public bool postProcessRequiresDepthTexture;
```

Comment:
```suggestion
        public bool postProcessingRequiresDepthTexture;
```

Reply:
Also needs public API docs. We have task to fix the ones missing atm.

Reply:
fixed in CS 79b78f0